pom.xml

```diff
@@ -17,6 +17,8 @@
 
         <!-- Plugins -->
         <jacoco-maven-plugin.version>0.8.5</jacoco-maven-plugin.version>
+        <mapstruct.version>1.3.1.Final</mapstruct.version>
```

Comment:
`<mapstruct-plugin.version>` and `<lombok-plugin.version>`. Just to easily differentiate dependencies and plugins